Philips.CodeAnalysis.Common/DiagnosticIds.cs

```diff
@@ -116,5 +116,6 @@ public enum DiagnosticIds
 		ThrowInformationalExceptions = 2121,
 		AvoidExceptionsFromUnexpectedLocations = 2122,
 		PassSenderToEventHandler = 2123,
+		DocumentUnhandledExceptions = 2124,

```

Comment:
PH2121 fired on this:
throw new ArgumentException($"The paths '{fromDirectory} and '{toPath}' have different path roots.");   [Setting "Requested Changes" for this one. Other feedback is at your judgement.]

Reply:
Nice catch. Added this as a test case and fixed the Analyzer